### ISSUE TITLE
fuse: unknown option(s): `-o use_ino,nonempty'

### ISSUE THREAD

**Issue by multiple1** [rpodgorny/unionfs-fuse#141]
Hello!
I am using the same mount options as in the example: https://github.com/rpodgorny/unionfs-fuse/blob/master/examples/S01a-unionfs-live-cd.sh#L6C42-L6C43

But I get this error using libfuse3.16.2:
```
fuse: unknown option(s): `-o use_ino,nonempty'
```
It seems that libfuse does no longer allow the `use_ino` mount option, because it should be set by the filesystem and not by the user.
https://github.com/libfuse/libfuse/commit/8ee553dac0297cfd75cbdd2d9cfdce37e22ef4ee

So am I safe to simply remove `use_ino` and unionfs-fuse handles this correctly?
I could not find use_ino in the unionfs source code, so I am not sure!

**Comment by bsbernd** [comment]
Sorry for late reply, I'm super busy. `use_ino` basically enables using the inode number provided by struct stat, instead of using a fuse internal consecutive counter. I think I had added that option to the examples/man-page as it helps to verify the right branch is used for a file. In the end neither fuse-internal inode number nor underlying file system inode (`use_ino`) are correct, as the inode number should be conserved during copy-on-write. And inode numbers also should be unique across different branches - for that the fuse internal inode number is even a little better.

**Comment by rpodgorny** [comment]
yeah, i've removed mentions of "use_ino" from examples and documentation. will be in next release.

**Comment by rpodgorny** [comment]
just released, closing
